### ISSUE TITLE
CIFAR10 baseline

### ISSUE THREAD

**Issue by bhat-prashant** [google-research/simclr#90]
1. In the paper, Linear Evaluation refers to Linear layer trained on top of frozen ResNet-50. However, in this repo, linear evaluation is done by fine tuning ResNet-50 layers (--fine_tune_after_block=4) . 

2. There's a little bit of confusion regarding cifar10 baseline. In this repo you have mentioned 91% while SimCLR paper report 95%. Please confirm.

3. The pytorch version of SimCLR (williamFalcon (PyTorch Lightning)) achieves only 86% test accuracy on CIFAR10. Please let me know from where does this difference stem from. 

Thank you for your time,
Prashant

**Comment by chentingpc** [comment]
> In the paper, Linear Evaluation refers to Linear layer trained on top of frozen ResNet-50. However, in this repo, linear evaluation is done by fine tuning ResNet-50 layers (--fine_tune_after_block=4) .

they're equivalent (as only linear layer is trained).
 
> There's a little bit of confusion regarding cifar10 baseline. In this repo you have mentioned 91% while SimCLR paper report 95%. Please confirm.

94% to be precise, that's using a ResNet-50 and trained for 1000epochs. The 91% is using ResNet-18 and has not been very well tuned.

> The pytorch version of SimCLR (williamFalcon (PyTorch Lightning)) achieves only 86% test accuracy on CIFAR10. Please let me know from where does this difference stem from.

I haven't checked their implementation so I cannot comment at this point.
